Sprint-3/implement/is-proper-fraction.js

```diff
@@ -32,3 +34,28 @@
 // target output: false
 // Explanation: The fraction 3/3 is not a proper fraction because the numerator is equal to the denominator. The function should return false.
 // These acceptance criteria cover a range of scenarios to ensure that the isProperFraction function handles both proper and improper fractions correctly and handles potential errors such as a zero denominator.
+//CHecked
+
+function isProperFraction(numerator, denominator){
+    if(denominator === 0){
+        return false;
+    }
+    else if(numerator === denominator){
+        return false;
+    }
+    else if(numerator === 0 && denominator === 0){
+        return false;
+    }
+    else if((Math.abs(numerator < 0)) && denominator > numerator){
+            return 'true';
```

Comment:
The boolean value 'true' does not need quotation marks

Reply:
removed the quotation marks